--- poppler/src/rbpoppler-page.c.orig	2010-12-30 08:08:55.000000000 +0100
+++ poppler/src/rbpoppler-page.c	2010-12-30 08:11:34.000000000 +0100
@@ -340,11 +340,11 @@
         poppler_page_get_size(page, &width, &height);
         rect.x2 = width;
         rect.y2 = height;
-        text = poppler_page_get_text(page,
+        text = poppler_page_get_selected_text(page,
                                      style,
                                      &rect);
     } else {
-        text = poppler_page_get_text(page,
+        text = poppler_page_get_selected_text(page,
                                      style,
                                      RVAL2POPPLER_RECT(rb_rect));
     }
